profiles/fabian4.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/60428924/153715844-9bdd713c-269f-47ba-bcba-fa3e821e46a9.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/153736827-018eaade-a0fd-414f-b641-2aae4c8463ef.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/153736827-018eaade-a0fd-414f-b641-2aae4c8463ef.png",
```